packages/grafana-data/src/transformations/transformers/ensureColumns.ts

```diff
@@ -1,25 +1,33 @@
+import { of } from 'rxjs';
+
 import { seriesToColumnsTransformer } from './seriesToColumns';
 import { DataFrame } from '../../types/dataFrame';
 import { getTimeField } from '../../dataframe/processDataFrame';
 import { DataTransformerInfo } from '../../types/transformations';
 import { DataTransformerID } from './ids';
+import { mergeMap } from 'rxjs/operators';
 
 export const ensureColumnsTransformer: DataTransformerInfo = {
   id: DataTransformerID.ensureColumns,
   name: 'Ensure Columns Transformer',
   description: 'Will check if current data frames is series or columns. If in series it will convert to columns.',
-  transformer: () => (data: DataFrame[]) => {
-    // Assume timeseries should first be joined by time
-    const timeFieldName = findConsistentTimeFieldName(data);
+  operator: (options = {}) => source =>
```

Comment:
nit: I think I would prefer having the explicit return type here to make things super obvious.

Reply:
I've used the same construct in most transformer operators, I don't think I'll change them all.